draft-ietf-quic-transport.md

```diff
@@ -943,47 +940,104 @@ The fields in the ACK frame are as follows:
+If the `N` bit is 1, the Num Blocks field indicates how many additional blocks
+follow the First Ack Block Length field.
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|[Num Blocks(8)]|      First Ack Block Length (8/16/32/48)    ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  [Gap 1 (8)]  |       [Ack Block 1 Length (8/16/32/48)]     ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  [Gap 2 (8)]  |       [Ack Block 2 Length (8/16/32/48)]     ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+                             ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  [Gap N (8)]  |       [Ack Block N Length (8/16/32/48)]     ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+{: #ack-block-format title="Ack Block Section"}
+
+The fields in the Ack Block Section are:
+
+* Num Blocks (opt): An optional 8-bit unsigned value specifying the number of
+  additional ack blocks (besides the required First Ack Block) in this ACK
+  frame.  Only present if the 'N' flag bit is 1.
 
-  * Num Blocks (opt): An optional 8-bit unsigned value specifying the number of
-    additional ack blocks (besides the required First Ack Block) in this ACK
-    frame.  Only present if the 'N' flag bit is 1.
+* First Ack Block Length: An unsigned packet number delta that indicates the
+  number of contiguous additional packets being acked starting at the Largest
+  Acked.
 
-  * First Ack Block Length: An unsigned packet number delta that indicates the
-    number of contiguous additional packets being acked starting at the Largest
-    Acked.
+* Gap To Next Block (opt, repeated): An unsigned number specifying the number
+  of contiguous missing packets from the end of the previous ack block to the
+  start of the next.
 
-  * Gap To Next Block (opt, repeated): An unsigned number specifying the number
-    of contiguous missing packets from the end of the previous ack block to the
-    start of the next.
+* Ack Block Length (opt, repeated): An unsigned packet number delta that
+  indicates the number of contiguous packets being acked starting after the
+  end of the previous gap.  Along with the previous field, this field is
+  repeated "Num Blocks" times.
 
-  * Ack Block Length (opt, repeated): An unsigned packet number delta that
-    indicates the number of contiguous packets being acked starting after the
-    end of the previous gap.  Along with the previous field, this field is
-    repeated "Num Blocks" times.
+### Timestamp Section {#timestamp-section}
+
+The Timestamp Section contains between zero and 255 measurements of packet
+delay. This information can be used by a sender to better estimate the RTT of
+the connection.
```

Comment:
The timestamps are simply packet receive times as observed by the receiver, and are not packet transit times. How about something like: "... of packet receive time deltas. This information may be used by a sender to better estimate the RTT of the connection."